__tests__/html/sendBox.buttonAlignment.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <script crossorigin="anonymous" src="/__dist__/testharness.js"></script>
+    <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
+    <style type="text/css">
+      #app {
+        height: 100%;
+      }
+
+      .webchat__send-box-text-box__input,
+      .webchat__send-box-text-box__html-text-area {
+        background-color: rgba(0, 0, 255, 0.6) !important;
```

Comment:
> `important!`

tsk tsk 😆

Reply:
This is correct CSS. It is `!important` instead of `important!`.

I don't exactly know why they make it that way, but the most compact form is as below, no space needed before `!` (exclamation mark).

```css
background-color: Red!important;
```

Reply:
My bad - ! at the end was a typo. I meant use of `!important` haha